1_fetch/src/get_nwis_data.R

```diff
@@ -1,30 +1,42 @@
-download_nwis_data <- function(site_num, parameterCd, startDate, endDate){
+download_nwis_data <- function(site_num, parameterCd, startDate, endDate, fileout){
 
   data_out <- readNWISdata(sites = site_num, parameterCd = parameterCd, startDate = startDate, endDate = endDate, service="iv")
-
+  # data_out <- data.frame(data_out)
```

Comment:
This isn't a dealbreaker, but readability is improved if you remove extraneous comments. Comments are super useful as debugging tools, for when you're swapping different code in and out - but when you commit your code it's best for that layer to be removed so that the comments are intentional pointers to help humans read the code.